tests/suites/cloud/test-app/containerA/Dockerfile.template

```
@@ -0,0 +1,3 @@
+FROM balenalib/%%BALENA_ARCH%%-alpine
+
+CMD bash -c "echo HELLO_WORLD; bash" 
```

```suggestion
CMD ["bash", "-c", "'echo HELLO_WORLD; bash'"] 
```
Nitpick ^ but nicer to look at. 
It's a double quote enclosed in single quotes